components/Sidebar.tsx

```diff
@@ -1,11 +1,14 @@
-const Sidebar = () => {
+import { ReactElement, ReactNode } from "react";
+
+type Params = {
+  children?: ReactNode;
+}
+
+const Sidebar = ({ children }: Params):ReactElement => {
   return (
-    <div className="w-1/5 bg-stone-900 text-white border-r-2">
-      <ul className="list-none flex flex-col items-center gap-3 pt-5">
-        <li className="hover:text-stone-400 cursor-pointer border-b-2 border-stone-400">Home</li>
-        <li className="hover:text-stone-400 cursor-pointer border-b-2 border-stone-400">About</li>
-        <li className="hover:text-stone-400 cursor-pointer border-b-2 border-stone-400">Contact</li>
-      </ul>
+    <div className="w-64 bg-[#222628] text-white flex flex-col border-[1px] border-[#a5a19950]">
```

Comment:
border-[1px] === border.
bg-[#222628] this maybe will be your primary color, and this border-[#a5a19950] maybe your primary border color, so add them to tw theme.